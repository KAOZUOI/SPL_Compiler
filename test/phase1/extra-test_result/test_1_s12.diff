1,3c1,3
< Error type B at Line 3: Missing semicolon ';'
< Error type B at Line 5: Missing semicolon ';'
< Error type B at Line 1: Missing semicolon ';' LOC3
---
> Error type B at Line 3 Missing semicolon ';'
> Error type B at Line 5 Missing semicolon ';'
> Error type B at Line 1 Missing semicolon ';'
5c5
< Error type B at Line 10: Missing semicolon ';'
---
> Error type B at Line 10 Missing semicolon ';'
7,9c7,9
< Error type B at Line 12: Missing semicolon ';'
< Error type B at Line 8: Missing semicolon ';' LOC2
< Error type B at Line 13: Missing semicolon ';' LOC1
---
> Error type B at Line 15: Missing closing parenthesis ')'
> Error type B at Line 22: Missing semicolon ';'
> Error type B at Line 28: Missing closing brace '}'
\ No newline at end of file
