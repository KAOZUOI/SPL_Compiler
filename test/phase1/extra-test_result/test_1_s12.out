Error type B at Line 3: Missing semicolon ';'
Error type B at Line 5: Missing semicolon ';'
Error type B at Line 1: Missing semicolon ';' LOC3
Error type B at Line 8: Missing closing parenthesis ')'
Error type B at Line 10: Missing semicolon ';'
Error type B at Line 12: Missing closing parenthesis ')'
Error type B at Line 12: Missing semicolon ';'
Error type B at Line 8: Missing semicolon ';' LOC2
Error type B at Line 13: Missing semicolon ';' LOC1
